src/demo/button/button.scss

```diff
@@ -3,11 +3,11 @@
 
 .demo-button {
   color: colors.$primary-brand;
-  transition: transform motion.$speed-quick motion.$ease-out;
 
   // TODO: These font styles are temporary, for demo purposes
   font-family: inherit;
   font-size: 18px;
+  transition: transform motion.$speed-quick motion.$ease-out;
```

Comment:
Is transition temporary? If not, I'd recommend changing the temporary comment to a footnote style to make it clearer which lines it applies to.

Reply:
Everything in `demo` is temporary.

Reply:
If it's temporary, then you're good to go. :)

Reply:
I just wanted to make sure that rule moving below the comment wasn't inadvertently confusing.